test/main/signers/hot/.userData/signers/3337653339623532373238346166633239303166663461353634363438666364.json

```diff
@@ -0,0 +1 @@
+{"id":"3337653339623532373238346166633239303166663461353634363438666364","addresses":["0xa663cf6642f7001b960aee1842bf44abf2833a0e"],"type":"ring","encryptedKeys":"a45af6b2a3a1a3adec24bf62e4ca705f:1cf4c51f51c539b1ee94054b25629916:d0b19914b5efa38b92d725b202beb096ce11bb695e154d6c33da832f30d64ae7084fe052ccb75081fc35f36362edc76a0d25252a19e56606795c3e339a169a36c2b440afeae4369f6c8cb89c74a8a96f"}
```

Comment:
these files dont need to be added to source control, I think they should be ignored actually

Reply:
Strange, I'm guessing i ran the tests but aborted half way through